spec/controllers/application_controller_spec.rb

```diff
@@ -1044,4 +1044,12 @@ def hub_path; end
       end
     end
   end
+
+  it "fails this test half the time" do
```

Comment:
Maybe we can rewrite this so that it succeeds if rspec is running failures, so it deterministically passes the second time.